collection-scripts/gather

```diff
@@ -79,6 +85,7 @@ display_help() {
 Usage: $0 [OPTIONS]
 
 Options:
+  -o,  --odf                Collect ODF logs (default operation mode)
```

Comment:
no need to mention about default operation mode, just ODF related resources and logs works